x-pack/plugins/ml/public/jobs/jobs_list/components/job_filter_bar/job_filter_bar.js

```diff
@@ -104,18 +127,27 @@ export class JobFilterBar extends Component {
         items: [
           {
             value: 'started',
-            name: 'Started'
+            name: intl.formatMessage({
+              id: 'xpack.ml.jobsList.jobFilterBar.startedLabel',
+              defaultMessage: 'Started'
+            })
           },
           {
             value: 'stopped',
-            name: 'Stopped'
+            name: intl.formatMessage({
+              id: 'xpack.ml.jobsList.jobFilterBar.stoppedLabel',
+              defaultMessage: 'Stopped'
+            })
           }
         ]
       },
       {
         type: 'field_value_selection',
         field: 'groups',
-        name: 'Group',
+        name: intl.formatMessage({
```

Comment:
As I understand it, we're not looking at CSS widths in this phase of work, but just logging the fact that these buttons will overflow the container with longer translated labels:

![image](https://user-images.githubusercontent.com/7405507/48557631-2ae80580-e8df-11e8-99d6-6b55ce0e0fcf.png)